src/main/java/io/electrum/vas/model/TransactionLabel.java

```diff
@@ -0,0 +1,66 @@
+package io.electrum.vas.model;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.Objects;
+
+import javax.validation.constraints.NotNull;
+
+@ApiModel(description = "A label assigned to a transaction by the entity processing the transaction. Labels are used for grouping or discovering transactions after transaction processing. Note that no transaction processing decision is made based on transaction labels.")
+public class TransactionLabel {
+
+   private String name;
+   private String value;
+
+   public void setName(String name) {
+      this.name = name;
+   }
+
+   public TransactionLabel name(String name) {
+      this.name = name;
+      return this;
+   }
+
+   public void setValue(String value) {
+      this.value = value;
+   }
+
+   public TransactionLabel value(String value) {
+      this.value = value;
+      return this;
+   }
+
+   @ApiModelProperty(required = true, value = "Label name used to uniquely identifier the transaction label.")
```

Comment:
```suggestion
   @ApiModelProperty(required = true, value = "Label name that indicates the type and intended use of the transaction label. The meaning of label names must be agreed upon by the parties involved in transaction processing.")
```
What do you think of this?